test/tests/types/basicTypegoose.test-d.ts

```diff
@@ -226,25 +226,31 @@ async function typeguards() {
   isRefType('string', String);
 
   // test errors
-  expectError<Parameters<typeof isDocument>[0]>({});
+  expect<Parameters<typeof isDocument>[0]>().type.not.toBeAssignable({});
```

Comment:
The expected error was: 'x is not assignable to y'. I would suggested using `.not.toBeAssignable()` for readability.